translations/strings.yaml

```diff
@@ -541,3 +541,12 @@ inAppAuth:
   termsOfServiceAndPrivacyDisclaimer:
     value: By continuing, you agree to the <a href="terms-of-service">Terms of Service</a> and the <a href="privacy-notice">Privacy Notice</a>.
     comment:  Do not remove or translate the markup '<a href="terms-of-service">', '<a href="privacy-notice">', '</a>'.
+
+#Accessibility
+accessiblity:
+  of:
+    value: of
```

Comment:
As a translator, I really wouldn't know how to translate this:
- We need a more descriptive ID that provides context about the string (and to prevent reuse in other situations). In this specific case, translating "of" is not enough, you need to expose the whole sentence (`x of y`?).
- Same for the next string. "Selected" needs to be tied to one element, as adjectives need to be declined (gender, number).